src/test/java/seedu/guest/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -26,7 +26,7 @@ public void setUp() {
     }
 
     @Test
-    public void execute_newPerson_success() {
+    public void execute_newGuest_success() {
         Guest validGuest = new PersonBuilder().build();
```

Comment:
Should this be GuestBuilder.build(), or should it not be refactored? Found in seedu.address.testutil.PersonBuilder

Reply:
This should be refactored under Zong Xian's (your) part as it is this under the testutil package.